scripts/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will move the pre-push hook from script folder to
+# the .git/hooks folder
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/setup.sh
+#
+# NOTE: this script should be run once after the initial codebase setup
+
+# Move file from script folder to .git/hooks folder
+cp scripts/pre-push.sh .git/hooks/pre-push
+
+# Download ktlint
+KTLINT="0.37.0"
```

Comment:
@anandwana001 
As there is issue with running ktlint in windows
can we switch ktlint version to `0.37.1` for prepush hook 
as in the `0.37.1` release they fixed the windows issue - https://github.com/pinterest/ktlint/issues/761

Reference - https://github.com/pinterest/ktlint/releases/tag/0.37.1

Reply:
(@Srikanth-Kb)

Reply:
@FareesHussain and @anandwana001 Thanks for the suggestions.
I will work on this.